### ISSUE TITLE
Can you tell me how to train this model? THANKS!!!

### ISSUE THREAD

**Issue by ytongW** [mcahny/Deep-Video-Inpainting#10]

**Comment by gonghuiyun** [comment]
我也在写这个训练模型，能留一个联系方式吗

**Comment by zhulei1228** [comment]
> 我也在写这个训练模型，能留一个联系方式吗

When can we get train code?thanks. @mcahny @gonghuiyun

**Comment by aryan461** [comment]
I tried to write the training code. But I need help with L_F and L_W losses. Could you please tell me how to define them using Flownet2? Or can you give us your losses code?

**Comment by zhulei1228** [comment]
> I tried to write the training code. But I need help with L_F and L_W losses. Could you please tell me how to define them using Flownet2? Or can you give us your losses code?

Have you trained with reconstruction loss?

**Comment by aryan461** [comment]
> Have you trained with reconstruction loss?
I didn't consider it. Did you do this? Does it work? so what about optical flow?

**Comment by mcahny** [comment]
Please check the "train_git.py " which can provide some reference algorithm, not yet runnable though.
Thanks.

**Comment by zhulei1228** [comment]
> Please check the "train_git.py " which can provide some reference algorithm, not yet runnable though.
> Thanks.
Can you release loss code?

**Comment by mcahny** [comment]
All loss functions (RECON_loss, HOLE_loss, ST_loss, LT_loss, flow_loss) are based on `nn.L1Loss()` , where RECON_loss includes extra ssim loss where I borrowed from [https://github.com/Po-Hsun-Su/pytorch-ssim/tree/master/pytorch_ssim](url)
